bin/printStream.py

```diff
@@ -65,7 +65,10 @@ def main():
     # Configure consumer connection to Kafka broker
     conf = {'bootstrap.servers': 'kafka:9092',
             'default.topic.config': {'auto.offset.reset': 'smallest'}}
-    conf['group.id'] = args.group
+    if args.group:
```

Comment:
You should also say what the default is when you add the group arg above, like you did in the `monitorStream.py`